install/kubernetes/cilium/templates/cilium-preflight/deployment.yaml

```diff
@@ -46,20 +46,20 @@ spec:
             cilium-dbg preflight validate-cnp;
             touch /tmp/ready-validate-cnp;
             sleep 1h;
-          livenessProbe:
-            exec:
-              command:
-              - cat
-              - /tmp/ready-validate-cnp
-            initialDelaySeconds: 5
-            periodSeconds: 5
           readinessProbe:
             exec:
               command:
               - cat
               - /tmp/ready-validate-cnp
-            initialDelaySeconds: 5
-            periodSeconds: 5
+            {{- if semverCompare "<1.20-0" .Capabilities.KubeVersion.Version }}
+            # The initial delay for the liveness probe is intentionally large to
```

Comment:
nit: this comment seems off? We use neither a liveness probe nor a startup probe.